### ISSUE TITLE
Create/Drop Policy Support

### ISSUE THREAD

**Issue by mariugul** [sqlfluff/sqlfluff#2015]
## Expected Behaviour
Should be able to create and drop policies using the PostgreSQL dialect.

`CREATE POLICY name ON table_name []...`
[Create Policy Documentation](https://www.postgresql.org/docs/10/sql-createpolicy.html)

`DROP POLICY [ IF EXISTS ] name ON table_name [ CASCADE | RESTRICT ]`
[Drop Policy Documentation](https://www.postgresql.org/docs/10/sql-droppolicy.html)

**Expected**:
`All Finished 📜 🎉!  `

## Observed Behaviour

```bash
$ sqlfluff lint --dialect postgres policy.sql 
== [policy.sql] FAIL                                                                              
L:   6 | P:   1 |  PRS | Line 6, Position 1: Found unparsable section: 'CREATE
                       | POLICY p1 ON person'
L:   7 | P:   1 |  PRS | Line 7, Position 1: Found unparsable section: 'DROP
                       | POLICY p1 ON person'
All Finished 📜 🎉!
```

## Steps to Reproduce

**policy.sql**:
```sql
CREATE TABLE person (
    id serial,
    name text
);

CREATE POLICY p1 ON person;
DROP POLICY p1 ON person;
```

**Lint policy.sql**:
```bash
sqlfluff lint --dialect postgres policy.sql 
```

## Dialect
`postgres`

## Version
`sqlfluff, version 0.8.2`
`Python 3.8.10`
`pip 20.0.2 `

## Configuration
```
--dialect postgres
```
